internal/selectors/selectors.go

```diff
@@ -179,6 +179,7 @@ func isExcluded(inst *unstructured.Unstructured) (bool, error) {
 	name := inst.GetName()
 	kind := inst.GetKind()
 	group := inst.GroupVersionKind().Group
+	// TODO(erikgb): This variable name collides with imported package name
 	labels := inst.GetLabels()
```

Comment:
`instLabels`? Though honestly I think this is only used once (line 194) so there's no need to make a local var for it.